src/components/OSCALDrawerSelector.test.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import OSCALDrawerSelector from "./OSCALDrawerSelector";
+
+describe("OSCALDrawerSelector", () => {
+  test("loads", () => {
+    render(<OSCALDrawerSelector open />);
+  });
+
+  test(`displays expected tree items`, () => {
+    render(<OSCALDrawerSelector open />);
+
+    const catalogItem = screen.getByRole("treeitem", {
+      name: "Catalog",
+    });
+
+    const componentItem = screen.getByRole("treeitem", {
+      name: "Component",
+    });
+
+    const profileItem = screen.getByRole("treeitem", {
+      name: "Profile",
+    });
+
+    const sspItem = screen.getByRole("treeitem", {
+      name: "System Security Plan",
+    });
```

Comment:
```suggestion
    const catalogItem = screen.getByRole("treeitem", {
      name: "Catalogs",
    });

    const componentItem = screen.getByRole("treeitem", {
      name: "Components",
    });

    const profileItem = screen.getByRole("treeitem", {
      name: "Profiles",
    });

    const sspItem = screen.getByRole("treeitem", {
      name: "System Security Plans",
    });
```

Okay so how much work would it take to make this work?